pycoal/environment.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import coal
+
+class EnvironmentalCorrelation:
+
+    def correlateMiningEnvironment(environmentFilename, miningFilename, outputFilename):
+
+        """
+        Correlate environmental impact GIS data with classified mines and save the result to a file.
+        """
+
+        # define function to select the environmental impact layer
+        def impactLayer(environmentDataset):
```

Comment:
@lewismc i indented the members of the class, so everything should be shifted. there might be an error though.